src/plugins/expressions/common/execution/execution.ts

```diff
@@ -213,6 +213,7 @@ export class Execution<
       },
       isSyncColorsEnabled: () => execution.params.syncColors,
       ...(execution.params as any).extraContext,
+      executionContext: execution.params.executionContext,
```

Comment:
should be a getter to prevent modification from within expressions

Reply:
I can prepend type with `readonly` to show the intent. IMO it's fine.

Reply:
ok, done